pkg/service/storage/ucan/replica_allocate.go

```diff
@@ -101,6 +104,7 @@ func ReplicaAllocate(storageService ReplicaAllocateService) server.Option {
 						Site:  cap.Nb().Site,
 						Cause: inv.Link(),
 					},
+					delegation.WithExpiration(ucan.Now()+transferTimeout),
```

Comment:
nit: what about making `const transferTimeout = 1 * time.Hour` and then

```suggestion
					delegation.WithExpiration(ucan.UTCUnixTimestamp(time.Now().Add(transferTimeout).Unix())),
```